t/native_cp_al/native_cp_al.t.in

```diff
@@ -0,0 +1,39 @@
+#!@PERL@
+use strict;
+use Test::More tests => 11; # pack here your amount of subtests
+use SysWrap;
+
+#
+# Test native_cp_al works with cmd_cp option not set and sync_first on (see contig).
+#
+
+ok(! remove_snapshot_root(),
+	" snapshot root does not exist before testing starts");
+
+ok(0 == rsnapshot("-c @TEST@/native_cp_al/conf/native_cp_al.conf sync 2>&1"),
+	" sync failed");
+ok(-d "@SNAP@/.sync/native_cp_al",
+	" sync failed");
+ok(0 == rsnapshot("-c @TEST@/native_cp_al/conf/native_cp_al.conf hourly 2>&1"),
+	" hourly backup after sync failed");
+ok(-d "@SNAP@/hourly.0/native_cp_al",
+	" hourly backup failed");
```

Comment:
I'm confused!

Reply:
Haha :/ It's been a while since I've written a Perl test case. I think I got my conventions mixed up. The text message should describe what happens in case of success right? OK. I review some of the other tests and they ain't bothering putting any description in there so I'll do the same.

Reply:
Yes, the text normally describes what *should* have happened.